src/muse.c

```diff
@@ -1829,6 +1831,17 @@ find_misc(struct monst* mtmp)
                         }
                     }
     }
+    if (OBJ_AT(mtmp->mx, mtmp->my) && !rn2(2)) {
+        for (otmp = g.level.objects[mtmp->mx][mtmp->my]; otmp; otmp = otmp2) {
+            otmp2 = otmp->nexthere;
+            if (Is_container(otmp) && otmp->otyp != BAG_OF_TRICKS
+                && Has_contents(otmp) && !otmp->olocked && !otmp->otrapped) {
+                g.m.has_misc = MUSE_BAG;
+                g.m.misc = otmp;
+                return TRUE;
+            }
+        }
+    }
```

Comment:
Does placing this above the `if (nohands(mdat)) return 0;` mean that monsters without hands can still open a container on their location?